src/module.c

```diff
@@ -38,6 +38,7 @@
 // Module-level global variables
 //------------------------------------------------------------------------------
 RG_Config config;                   // Module global configuration.
+void (*on_config_change)(Config_Option_Field type, void *val); // callback function which called when config param changed
```

Comment:
Define a type for this function signature,
the declaration of this type should be in `config.h`